src/styles/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  width: 200px;
+  border: 1px solid white;
+  box-sizing: border-box;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__title {
+    margin: 40px 0 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__sudtitle {
+    margin: 4px 0 0;
+    padding-top: 0;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    font-size: 10px;
+  }
```

Comment:
It appears there is a typo in the BEM element name `&__sudtitle`. It should be `&__subtitle` to follow the proper naming convention and maintain consistency.